internal/pkg/fileutil/fileutil.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package fileutil
+
+import (
+	"os"
+	"time"
+)
+
+// FileExists returns true if file/dir exists
+func FileExists(fp string) (ok bool, err error) {
+	if _, err := os.Stat(fp); err == nil {
+		ok = true
+	} else if os.IsNotExist(err) {
```

Comment:
```suggestion
	} else if errors.Is(err, fs.ErrNotExist) {
```

you would also need to import `errors` and `io/fs`